pkg/proxy/iptables/proxier.go

```diff
@@ -1572,7 +1572,17 @@ func (proxier *Proxier) syncProxyRules() {
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
 		glog.Errorf("Failed to execute iptables-restore: %v", err)
-		glog.V(2).Infof("Rules:\n%s", proxier.iptablesData.Bytes())
```

Comment:
Printing the rules is only useful if the error has to do with a specific rule (as opposed to "could not get xtables lock" or something like that). But the code to generate the rules only has so many different cases in it and none of them are super obscure, so any bug in that code would get hit by developers pretty quickly rather than escaping into a release.

So, printing the rules is probably only ever going to be useful on non-production systems. So just change it from V(2) to V(4).